src/main/resources/banner.txt

```diff
@@ -1,13 +1,14 @@
-_  _ _  _ ___  ___  ____  
-|__| |\/| |__] |__] [__   
-|  | |  | |    |    ___]  
-                          
-___ ____ _  _ ___  _    ____ ___ ____ 
- |  |___ |\/| |__] |    |__|  |  |___ 
- |  |___ |  | |    |___ |  |  |  |___ 
-                                      
-_  _ ____ ___ _    _ _  _ 
-|_/  |  |  |  |    | |\ | 
-| \_ |__|  |  |___ | | \| 
-                          
-Please change me by generating your own ASCII art and placing in banner.txt
+
+██████╗░██████╗░██╗░██████╗░█████╗░███╗░░██╗███████╗██████╗░  ███████╗██████╗░░█████╗░███╗░░░███╗
+██╔══██╗██╔══██╗██║██╔════╝██╔══██╗████╗░██║██╔════╝██╔══██╗  ██╔════╝██╔══██╗██╔══██╗████╗░████║
+██████╔╝██████╔╝██║╚█████╗░██║░░██║██╔██╗██║█████╗░░██████╔╝  █████╗░░██████╔╝██║░░██║██╔████╔██║
+██╔═══╝░██╔══██╗██║░╚═══██╗██║░░██║██║╚████║██╔══╝░░██╔══██╗  ██╔══╝░░██╔══██╗██║░░██║██║╚██╔╝██║
+██║░░░░░██║░░██║██║██████╔╝╚█████╔╝██║░╚███║███████╗██║░░██║  ██║░░░░░██║░░██║╚█████╔╝██║░╚═╝░██║
+╚═╝░░░░░╚═╝░░╚═╝╚═╝╚═════╝░░╚════╝░╚═╝░░╚══╝╚══════╝╚═╝░░╚═╝  ╚═╝░░░░░╚═╝░░╚═╝░╚════╝░╚═╝░░░░░╚═╝
+
+███╗░░██╗░█████╗░███╗░░░███╗██╗░██████╗  ███╗░░░███╗██╗░██████╗░██████╗░░█████╗░████████╗██╗░█████╗░███╗░░██╗
+████╗░██║██╔══██╗████╗░████║██║██╔════╝  ████╗░████║██║██╔════╝░██╔══██╗██╔══██╗╚══██╔══╝██║██╔══██╗████╗░██║
+██╔██╗██║██║░░██║██╔████╔██║██║╚█████╗░  ██╔████╔██║██║██║░░██╗░██████╔╝███████║░░░██║░░░██║██║░░██║██╔██╗██║
+██║╚████║██║░░██║██║╚██╔╝██║██║░╚═══██╗  ██║╚██╔╝██║██║██║░░╚██╗██╔══██╗██╔══██║░░░██║░░░██║██║░░██║██║╚████║
+██║░╚███║╚█████╔╝██║░╚═╝░██║██║██████╔╝  ██║░╚═╝░██║██║╚██████╔╝██║░░██║██║░░██║░░░██║░░░██║╚█████╔╝██║░╚███║
+╚═╝░░╚══╝░╚════╝░╚═╝░░░░░╚═╝╚═╝╚═════╝░  ╚═╝░░░░░╚═╝╚═╝░╚═════╝░╚═╝░░╚═╝╚═╝░░╚═╝░░░╚═╝░░░╚═╝░╚════╝░╚═╝░░╚══╝
```

Comment:
I thought this font looked cool .. but found it just displays in ???s in my local console !

Reply:
Weird that works for me

Reply:
I wonder if it is a Windows thing